src/AppCommon/AppCommon.csproj

```diff
@@ -1,24 +1,22 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
So we are also updating to net8, which means we also need to update https://docs.particular.net/nservicebus/throughput-tool/#installation-net-tool-recommended

Reply:
yep - https://github.com/Particular/docs.particular.net/pull/6701